### ISSUE TITLE
Consistent letter casing throughout the codebase

### ISSUE THREAD

**Issue by pulkitvyas08** [ChiragSaxena5437/PARiKSHA#46]
Currently, the letter casing is very inconsistent some parts of code have snake_case, some have camelCase, some have PascalCase and there there are the ones with runningcase
Fix all that once all the important work is done

**Comment by pulkitvyas08** [comment]
This was more or less taken care of in the later phase of the project so closing for now
